docs/4.3/enterprise/quickstart-enterprise.md

```diff
@@ -41,6 +41,11 @@ _"node.example.com"_  | 10.1.1.11      | This server will only run the SSH servi
 This Quick Start Guide assumes that both servers are running a [systemd-based](https://www.freedesktop.org/wiki/Software/systemd/)
 Linux distribution such as Debian, Ubuntu or a RHEL derivative.
 
+## Optional: Quickstart using Docker
```

Comment:
Longer term we'll split this out, but I think it's ok to add into the quickstart for now.